signer/plugin.go

```diff
@@ -52,12 +52,6 @@ func (s *pluginSigner) Sign(ctx context.Context, desc ocispec.Descriptor, opts n
 	if err != nil {
 		return nil, nil, err
 	}
-	if !metadata.SupportsContract(proto.ContractVersion) {
```

Comment:
Since #200 was merged recent, could you scan the package `signer` for updates like removing `ContractVersion: proto.ContractVersion` in the request?

Reply:
Updated.